python/splitter.py

```diff
@@ -54,9 +71,9 @@ def __main__():
     # tip = get_tip_amount(tip, subtotal)
     # tax = percentage_helper(tax, subtotal)
 
-    participants = input("How many participants are there? --> ")
-    subtotal = input("SUBTOTAL: What is the pre-tax and pre-tip subtotal? --> ")
-    final_cost = input("FINAL COST: What is the amount paid after all fees, taxes, and tip? --> ")
+    participants = validate_input("How many participants are there? --> ")
```

Comment:
Do you want to validate this a a float?

Reply:
That's a good point, although this is probably the only place that would actually require an integer. I can probably just validate that separately or create another function for validating integers in case it comes up in other places. Could get confusing if there's multiple really similar functions, like what you already mentioned about is_number()

Reply:
I mean this is not wrong, i just wanted to know if this is what you intended, if so thats fine, validating if its a float will still work if you put an int but we probably don't want it to work if you put a float. ex 2.5 users will break the code further down